lib/ansible/modules/network/meraki/meraki_network.py

```diff
@@ -206,6 +223,9 @@ def main():
             meraki.fail_json(msg='net_name or net_id is required for present or absent states')
     if meraki.params['net_name'] and meraki.params['net_id']:
         meraki.fail_json(msg='net_name and net_id are mutually exclusive')
+    if not meraki.params['net_name'] and not meraki.params['net_id']:
+        if meraki.params['enable_vlans']:
+            meraki.fail_json(msg='enable_vlans parameter requires net_name or net_id to be specified')
```

Comment:
So something I tend to do is make messages easy to understand by using quotes around names and values.
```suggestion
            meraki.fail_json(msg="The parameter 'enable_vlans' requires 'net_name' or 'net_id' to be specified")
```